server_admin/topics/identity-broker/social/openshift.adoc

```diff
@@ -37,3 +37,48 @@ Use client ID and secret defined by `oc create` command to enter them back on th
 Go back to {project_name} and specify those items.
 
 Please refer to https://docs.okd.io/latest/architecture/additional_concepts/authentication.html#oauth[official OpenShift documentation] for more detailed guides.
+
+==== OpenShift 4
+
+NOTE: Prior to configuring OpenShift 4 Identity Provider, please locate the correct OpenShift 4 API URL up.
+In some scenarios, that URL might be hidden from users. The easiest way to obtain it is to invoke the following command `curl -s -k -H "Authorization: Bearer $(oc whoami -t)" https://<openshift-user-facing-api-url>/apis/config.openshift.io/v1/infrastructures/cluster | jq ".status.apiServerURL"`. In most cases, the address will be protected by HTTPS. Therefore, it is essential to configure `X509_CA_BUNDLE` in the container and set it to `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`. Otherwise, {project_name} won't be able to communicate
```

Comment:
Are you sure the CA certificate is placed in `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`? It seems to be true for OpenShift 3 but I couldn't find anything about this for OpenShift 4.

Reply:
Thanks for the comments @vmuzikar !

* Link shouldn't be clickable anymore.
* I added a short note that a user might need to install `jq` separately
* Yes, the `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt` is the proper one (see the [docs](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod). It may happen that your cluster doesn't have a CA and therefore there is no such file. However, RHSSO (with the latest changes that are still in PR) can tolerate empty files, so it shouldn't be an issue.